compiler/noirc_frontend/src/node_interner.rs

```diff
@@ -1429,6 +1433,19 @@ impl NodeInterner {
     pub(crate) fn ordering_type(&self) -> Type {
         self.ordering_type.clone().expect("Expected ordering_type to be set in the NodeInterner")
     }
+
+    fn try_resolve_trait_impl_location(&self, location: Location) -> Option<Location> {
```

Comment:
This looks fine to me but could use a comment to point out we're retrieving the trait's location from the location of the trait name in an impl.

Reply:
bc85372: chore: adds doc comment